sdk/core/core-lro/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 2.6.1 (Unreleased)
+## 3.0.0-beta.1 (2024-02-25)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+Initial implementation of synchronized LRO
```

Comment:
what does synchronized LRO mean in the context of this change?

Reply:
I updated the changelog entry as following:
```md

## 3.0.0-beta.1 (2024-02-25)

Initial implementation of next-generation for Long Running Operations (LROs) in which we deprecate the `LroEngine` support and change poller creation to a synchronized way.

- `LroEngine` is removed and no long supported
- The return type of `createHttpPoller` is changed from `Promise<SimplePollerLike>` to `PollerLike`
- Some interfaces are renamed. `SimplePollerLike` is renamed as `PollerLike`, `LroResponse` is renamed as `OperationResponse` and `LroResourceLocationConfig` is to `ResourceLocationConfig`
- Functions `getOperationState()`, `getResult()`, `isDone()` and `isStopped()` is changed to read-only attributes `operationState`, `result`, `isDone` and `isStopped`
- Removed the attributes `requestMethod` and `requestPath` in `LongRunningOperation`
- Added a new function `serialize` to help serialize the poller
- Added a new function `submitted` to help wait for the poller submitted succesffully
- Added a new parameter `TRequest` for `OperationResponse` to accept the raw request
- Export the function `deserializeState` to the public
```